src/test/java/edu/harvard/iq/dataverse/api/ExternalToolsIT.java

```diff
@@ -145,26 +130,14 @@ public void testFileLevelTool1() {
                 .statusCode(OK.getStatusCode())
                 // No tools for this file type.
                 .body("data", Matchers.hasSize(0));
+        
+        //Delete the tool added by this test...
+        Response deleteExternalTool = UtilIT.deleteExternalTool(toolId);
```

Comment:
@sekmiller for the other delete, you follow with an:
     deleteExternalTool.then().assertThat()
                .statusCode(OK.getStatusCode());
For consistency can you add that here?

Reply:
sure